src/components/section-list.component.js

```diff
@@ -66,47 +71,42 @@ export const SectionList = ({
 
   if (loading) {
     listDisplay = (
-      <ActivityIndicator animating={loading} style={styles.loadingIcon} />
+      <LoadingIcon animating={loading}
+      />
     );
   } else if (noItems) {
     listDisplay = (
-      <ListItem
-        title={noItemsMessage}
-        titleStyle={styles.listTitle}
-        hideChevron
-      />
+      <SectionListItem title={noItemsMessage} hideChevron />
     );
   } else {
     listDisplay = children;
   }
 
   let sectionTitle = '';
-
   if (typeof title === 'string') {
-    sectionTitle = <Text style={styles.sectionTitle}>{title}</Text>;
+    sectionTitle = <SectionTitle>{title}</SectionTitle>;
   } else {
-    sectionTitle = <View style={{ padding: 15 }}>{title}</View>;
+    sectionTitle = <TitleView>{title}</TitleView>;
   }
 
   return (
-    <View style={styles.section}>
-      <View style={styles.topHeader}>
+    <Section>
+      <Header>
         {sectionTitle}
 
         {showButton &&
           !loading && (
-            <Button
+            <StyledButton
               title={buttonTitle}
               textStyle={fonts.fontPrimarySemiBold}
               fontSize={13}
               color={showButton ? colors.primaryDark : colors.white}
-              buttonStyle={styles.button}
```

Comment:
Besides `buttonStyle`, please move other props like `textStyle/fontSize/color` to styled-components attrs.